docs/Introduction/architecture-overview.md

```diff
@@ -22,7 +22,7 @@ To learn how to make a GET request using a single oracle, see [Make a GET Reques
 
 # Decentralized Data Model
 
-For a more robust and trustworthy answer, you can aggregate data from many oracles. With on-chain aggregation, data is aggregated from a decentralized network of independent oracle nodes. This architecture is applied to Chainlink Price Feeds, which aggregate asset price data.
+For a more robust and trustworthy answer, you can aggregate data from many oracles. With on-chain aggregation, data is aggregated from a decentralized network of independent oracle nodes. This architecture is applied to Chainlink Data Feeds, which aggregate asset price data.
```

Comment:
```suggestion
For a more robust and trustworthy answer, you can aggregate data from many oracles. With on-chain aggregation, data is aggregated from a decentralized network of independent oracle nodes. This architecture is applied to Chainlink Data Feeds, which can aggregate data such as asset price data.
```

Search and replace works for most of these changes, but some of the sentences make less sense when we include both "price" and "data" in the same sentence.

Conceptually, we want customers to think of data feeds as the main category, and price feed data as a type of data that Chainlink Data Feeds can work with.